### ISSUE TITLE
Lbryum cannot handle decimal.Decimal types in command line or when running as daemon 

### ISSUE THREAD

**Issue by kaykurokawa** [lbryio/lbryum#146]
When running lbryum as a daemon, it cannot handle decimal types because the JSON RPC server that it uses does not handle converting decimal types to JSON. 

When running lbryum through the command line, it cannot handle decimal types because it uses the standard json.dumps() function. 

(Note that lbrynet uses lbryum as a library so it does not have either of these problems)

**Comment by kauffj** [comment]
This doesn't seem like a priority since it is only the lbryum daemon, correct me if I'm wrong.

**Comment by eukreign** [comment]
Once lbryum is merged into lbrynet this will be even less of a problem (or non-existent).
